roles/kubernetes-apps/ansible/templates/nodelocaldns-config.yml.j2

```diff
@@ -17,7 +17,7 @@ data:
         loop
         bind {{ nodelocaldns_ip }}
         forward . {{ block['nameservers'] | join(' ') }}
-        prometheus :9253
+        prometheus {% if nodelocaldns_bind_metrics_host_ip %}{$MY_HOST_IP}{% endif %}:9253
```

Comment:
I had a question about the syntax `{$FOO}`, but that seems fine from https://coredns.io/manual/configuration/#environment-variables

Reply:
Yes, that is the way `Caddyfiles` (the configuration file format used by CoreDNS) do interpolation.